mkt/websites/models.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from django.conf import settings
+from django.db import models
+
+import json_field
+
+from mkt.site.models import ModelBase
+from mkt.translations.fields import TranslatedField
+from mkt.tags.models import Tag
+from mkt.translations.fields import save_signal
+
+
+class Website(ModelBase):
+    default_locale = models.CharField(max_length=10,
+                                      default=settings.LANGUAGE_CODE)
+    url = TranslatedField()
+    title = TranslatedField()
+    short_title = TranslatedField()
+    description = TranslatedField()
+    keywords = models.ManyToManyField(Tag)
+    # FIXME regions
+    submitter_description = TranslatedField()
+    categories = json_field.JSONField(default=None)
+    icon_type = models.CharField(max_length=25, blank=True)
+    icon_hash = models.CharField(max_length=8, blank=True)
+    # FIXME devices
+    last_updated = models.DateTimeField(db_index=True, auto_now_add=True)
+    # FIXME submitter
```

Comment:
Copy `AddonUser` mechanism ? Call it `WebsiteSubmitter` ? Or just make a generic M2M ? (PRD says *"We want to track the other Submitters who submitted the same URL and “attach” them to the website."*)